test/common/model/fields.spec.js

```diff
@@ -0,0 +1,90 @@
+const chai = require('chai');
```

Comment:
Do we not want to nest this file under an `hca` directory in the `test` directory? Same with a few of the other test files here.